docs/en/latest/concepts/annotations.md

```diff
@@ -360,3 +360,32 @@ spec:
             port:
               number: 50053
 ```
+
+## Cross namespace reference service
```

Comment:
Maybe it can just be "Cross-namespace reference"?

Reply:
The "cross-namespace references" include objects such as secret. In fact, only service is supported here.

Reply:
Then that could be mentioned in the description? "cross-namespace reference service" does not convey the meaning that it is cross-namespace reference for services. Hence I suggested just cross-namespace reference. When a user then reads the description it will be clear that this annotation is used to reference services.

Reply:
done.